solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -1,3 +1,30 @@
+.liquidity-banner {
+  margin: 0.8rem 3.2rem;
+  padding: 0.8rem 0;
+  width: auto;
+  min-width: 25rem;
+  max-width: 100%;
+  display: flex;
+
+  svg.liquidity-banner__keep-logo {
+
+    height: 2rem;
+    width: 2rem;
+    margin: 0 1rem;
+  }
+
+  .liquidity-banner__content {
+    .liquidity-banner__title{
+      font-size: 1.1rem;
```

Comment:
As I can see on Figma, `font-size` and `font-weight` are unnecessary here.

[Ttitle styles in Figma](https://www.figma.com/file/Su61MwHLrQn3ld24q85pMx/Token-Dashboard-dApp?node-id=8175%3A13579):
```
font-family: Work Sans;
font-style: normal;
font-weight: normal;
font-size: 16px;
line-height: 24px;
```